music21/stream/__init__.py

```diff
@@ -13349,10 +13338,6 @@ def makeNotation(self,
                                inPlace=True,
                                bestClef=bestClef,
                                **subroutineKeywords)
-            # note: while the local-streams have updated their caches, the
-            # containing score has an out-of-date cache of flat.
-            # this, must call elements changed
-            returnStream.coreElementsChanged()
         else:  # call the base method
```

Comment:
Can we leave this back in?  I remember that this was a very subtle error that was hard to diagnose, and I'd rather have it than lose another day trying to figure out what happened.  Can add "Probably not needed."